ecs-cli/integ/cmd_up/up_gpu_test.go

```diff
@@ -0,0 +1,69 @@
+// +build integ
+
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cmd_up
+
+import (
+	"fmt"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/integ"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// TestCusterCreationWithGPUInstances runs the 'ecs-cli up -c <clusterName> --instance-type <gpuInstance> --capability-iam --force' command.
+func TestCusterCreationWithGPUInstances(t *testing.T) {
+	// Given
+	cfnClient, ecsClient, clusterName := setup(t)
```

Comment:
My understanding is that `clustername` will contain a GUID to distinguish it apart from other clusters created in the same test run. Might also be helpful (perhaps for debugging) to add an identifier for which test its created for - E.g., "gputest" or "uptest" - that we could pass in here. What do you think?

Reply:
Correct, the `SuggestedResourceName` function took optional identifiers that does what you're suggested. I've changed it to just a string instead.